src/Main/Main.jsx

```diff
@@ -6,10 +6,17 @@ import classnames from "classnames";
 import styles from "./Main.module.scss";
 
 export function Main(props) {
-	const { children, className, ...rest } = props;
+	const { children, className, withPadding, ...rest } = props;
```

Comment:
Should `withPadding` default to `true`? I think the most common scenario is that we _do_ want padding, so it'd be 'opt out' if you had wanted to remove it?

```suggestion
	const { children, className, withPadding = false, ...rest } = props;
```

Reply:
can you double check I haven't misunderstood the comment above please Ian? I've defaulted to true on line 9...